### ISSUE TITLE
Search can't find a module

### ISSUE THREAD

**Issue by brunocalado** [sdenec/tidy-ui_game-settings#18]
There is a module which can't be found using the search. You have to manually find it.

Can you take a look?

https://github.com/Brownie79/swade-macros-simple

**Comment by sdenec** [comment]
The module's name was little strange with all lowercase and dashes. Normally all names start with an uppercase letter and have separate words. Threw the search string off. Should be fixed now! Thanks for pointing this out.

**Comment by ccjmk** [comment]
I think there might still be an issue of somewhat erratic searches. I noticed that it doesn't matter if you type lowercase, uppercase or Exact Case when searching a module.. like, all wait/WAIT/Wait/waIT will find "Hey, Wait!"

But on a quick trial-and-error, I noticed some modules where not found (might be related to reserved words?)

* "libWrapper" was never found, with LIB, lib, Lib, l(lowercase L), L, but anything with W/Wra/wra etc works.
* "GM Notes" is found if I type G, not GM/gm
* "Jitsi WebRCT client" can't be found by anything trying to reference client (just a C works, but anything cl.* fails.
